tools/issue-labeler/Directory.Build.props

```diff
@@ -1,5 +1,12 @@
+
 <Project>
 
+    <PropertyGroup>
```

Comment:
Do we need this in the tools repo?   I don't recall there being any redirection globally.